lib/cli/commands/listtrades.ts

```diff
@@ -0,0 +1,45 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient } from '../command';
+import { ListTradesRequest, ListTradesResponse, Trade } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+
+const HEADERS = [
+  colors.blue('rhash'),
```

Comment:
I don't think we really need the payment hash on the prettified, user-friendly table. Thoughts @kilrau?

If we do keep it, I'd probably call this "Payment hash" or maybe "Swap hash".

Reply:
removed it for now, please let me know if you want to put it back